src/index.ts

```diff
@@ -8,26 +8,39 @@ import { Scenario } from './model/Scenario';
 import { TestResult } from './model/TestResult';
 import { loadAllScenarios, loadScenariosById } from './scenarioLoading';
 import { loadYamlConfiguration } from './yamlParsing';
+import { stringify } from 'querystring';
```

Comment:
imports from dependencies should be placed before local imports, preferrably in alphabetical order

Reply:
passt